### PR TITLE
Update chain registration script to include Sui

### PR DESCRIPTION
**Pull request by heyitaki**
This PR adds Sui to registration script and sneaks in a cleanup change (removing hardcoded Aptos consts in clients/js)

### CODE REVIEW

File: clients/js/main.ts [hunk 1]
```diff
@@ -45,37 +57,6 @@ const GOVERNANCE_CHAIN = 1;
 const GOVERNANCE_EMITTER =
   "0000000000000000000000000000000000000000000000000000000000000004";
 
-// TODO: remove this once the aptos SDK changes are merged in
```
Comment (kcsongor) [inline clients/js/main.ts:48]: 🙏

File: clients/js/main.ts [hunk 2]
```diff
@@ -920,16 +901,13 @@ function parseAddress(chain: ChainName, address: string): string {
     return "0x" + hex(address).substring(2).padStart(64, "0");
   } else if (chain === "osmosis") {
     throw Error("OSMOSIS is not supported yet");
-  } else if (chain === "sui") {
-    throw Error("SUI is not supported yet");
-  } else if (chain === "aptos") {
-    if (/^(0x)?[0-9a-fA-F]+$/.test(address)) {
-      return "0x" + evm_address(address);
+  } else if (chain === "sui" || chain === "aptos") {
```
Comment (kcsongor) [inline clients/js/main.ts:904]: does it apply to sui as well?

**Review by kcsongor — APPROVED** [review]

File: clients/js/main.ts (hunk 2 above)
Comment (heyitaki) [reply clients/js/main.ts]: ah the fully qualified part would not, pushed a fix

**Review by jumpsiegel — APPROVED** [review]

**Comment by evan-gray** [comment]
Moving to draft, I think we want these changes into the `sui/tilt` branch. Might need to rewrite.

**Review by bruce-riley — APPROVED** [review]

**Comment by heyitaki** [comment]
Closing in favor of #2605

**Pull request closed without merging**